test/e2e/testing-manifests/storage-csi/external-snapshotter/volume-group-snapshots/csi-hostpath-plugin.yaml

```diff
@@ -276,7 +276,7 @@ spec:
               mountPath: /csi
 
         - name: node-driver-registrar
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.10.1
+          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.13.0
```

Comment:
I think changes to the CSI sidecars should be driven and reviewed by SIG Storage.

I can't tell in this case (too long out of it), but testing a diverse range of sidecar versions at some point was part of the test plan. Older sidecar versions are supported with current Kubernetes, so they have to be tested, not just "the latest" or "the same one".

Reply:
cc @jsafrane @msau42 @xing-yang

Reply:
Hmm, I don't think k/k should be responsible for testing a big range of a project like this, but the tests could be re-used by configuring an override or something.

Reply:
> Older sidecar versions are supported with current Kubernetes, so they have to be tested, not just "the latest" or "the same one".

... is this written somewhere? My understanding was that CSI advertises recommended version / compatibility with Kubernetes, not the other way around.

If Kubernetes is committing to support some sets of versions, this needs to be a visible policy.